### ISSUE TITLE
Links are broken in dynamic blocks when the task has the cookies [/] or [%]

### ISSUE THREAD

**Issue by maikol-solis** [alphapapa/org-ql#248]
According to the org-mode docs (https://orgmode.org/manual/Breaking-Down-Tasks.html) we can add cookies to split tasks into subtasks. 

However, the dynamic block broke the links for all the tasks having these cookies, because it adds the information of the cookie within the link.  

In the example below, in the dynamic block, if you click in the `Task 1`, it doesn't go to anywhere because the right link should be  `[[Task 1][Task 1​]] `

```
#+BEGIN: org-ql :query (and (planning :from "2021-08-21" :to "2021-08-30") (todo)) :columns (todo (priority "P") deadline heading)  :ts-format "%Y-%m-%d %H:%M"
| Todo | P | Deadline         | Heading                          |
|------+---+------------------+----------------------------------|
| TODO | B | 2021-08-23 00:00 | [[Task 1 \[1/2\]][Task 1 [1/2]​]] |
| TODO |   | 2021-08-27 00:00 | [[Subtask 2][Subtask 2]]         |
| TODO | A | 2021-08-27 00:00 | [[Task 2][Task 2]]               |
#+END:

* TODO [#B] Task 1 [1/2]
DEADLINE: <2021-08-23 Mon>
** DONE Subtask 1
DEADLINE: <2021-08-24 Tue>
** TODO Subtask 2
DEADLINE: <2021-08-27 Fri>
* TODO [#A] Task 2
DEADLINE: <2021-08-27 Fri>

```

Thanks.

**Comment by alphapapa** [comment]
Oops, I hadn't noticed that.  Thanks.

I'll get to this, but I have some other things going on too, and it shouldn't be too hard of a fix, so I'll tag this as "help wanted" and such, in case anyone else would like to pitch in.  :)

**Comment by novoid** [comment]
Is there a workaround where I get `id:foo` (with the appropiate `:PROPERTY:` id) in a column of the dynamic table? 

Currently, [all of my projects do have progress indicator cookies](https://karl-voit.at/2019/11/03/org-projects/) and therefore I can't use org-ql for generating my project overview and jump table ...

**Comment by alphapapa** [comment]
@maikol-solis @novoid A simple test shows that it seems to be fixed now.  Please let me know if it works for you.  Thanks.

**Comment by alphapapa** [comment]
Yes, it won't be fixed on MELPA until the package is rebuilt.

**Comment by maikol-solis** [comment]
> @maikol-solis @novoid A simple test shows that it seems to be fixed now. Please let me know if it works for you. Thanks.

Glad to help!

**Comment by novoid** [comment]
Today I updated my Elpa packages and got `org-ql-20220301.841` which fixed the issue. This is very much welcomed as the last step to get rid of (very very slow) default dynamic tables for a project overview by switching to fast and elegant `org-ql`. Thank you! :bow:

**Comment by alphapapa** [comment]
Rebased this fix into 0.6.1 stable release rather than 0.7-pre.
